src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -65,7 +66,8 @@ public NodeComments getComments() {
   public void process(Node externs, Node root) {
     for (Node script : root.children()) {
       if (script.isScript()) {
-        List<Comment> comments = compiler.getComments(script.getSourceFileName());
+        ImmutableList<Comment> comments =
```

Comment:
Yes, sorry, I might be a bit too familiar with the guava collections; I wanted to signal that this isn't intended to change.  Adding a comment about the no-copy, copy.